README.md

```diff
@@ -66,10 +66,11 @@ If you’d prefer to compile from source code please follow [these instructions]
 
 ## Compatibility
 
-Lightning Terminal is backwards compatible with `lnd` back to version v0.12.0-beta.
+Lightning Terminal is backwards compatible with `lnd` back to version v0.13.0-beta.
```

Comment:
We're still backward compatible to `lnd v0.12.0-beta` as we currently pull in the `github.com/lightninglabs/lndclient v0.12.0-8` dependency. I assume we don't want to break this backward compatibility for remote `lnd` users?